skbio/embedding/_embedding.py

```diff
@@ -98,3 +108,158 @@ def __repr__(self):
+
+        seq = np.array([sequence], dtype='O')
+        super(SequenceVector, self).__init__(vector, seq, **kwargs)
+
+    def __str__(self):
+        return str(self._ids[0].decode('ascii'))
+
+    @property
+    def sequence(self):
+        r""" String representation of the underlying sequence """
+        return str(self)
+
+    @property
+    def vector(self):
+        return self.embedding.squeeze()
+
+    def __repr__(self):
+        r"""
+        Return a string representation of the SequenceEmbedding object.
+
+        Returns
+        -------
+        str
+            A string representation of the SequenceEmbedding object.
+
+        See Also
+        --------
+        Protein
+        """
+        seq = Sequence(str(self))
+
+        rstr = repr(seq)
+        rstr = rstr.replace("Sequence", "SequenceVector")
+        n_indent = 4  # see Sequence.__repr__
+        indent = " " * n_indent
+        dim = self.embedding.shape[1]
+        rstr = rstr.replace(
+            "has gaps",
+            f"embedding dimension: {dim}\n{indent}has gaps",
+        )
+        return rstr
+
+    @property
+    def embedding(self):
+        r""" The embedding tensor. """
+        return self._embedding.reshape(1, -1)
+
+    @staticmethod
+    def to_numpy(sequence_vectors : List["SequenceVector"]):
+        lens = [len(pv.vector) for pv in sequence_vectors]
+        if not all(ln == lens[0] for ln in lens):
+            raise ValueError("All vectors must have the same length.")
+        data = np.vstack([pv.vector for pv in sequence_vectors])
+        return data
+
+    @staticmethod
+    def to_distance_matrix(sequence_vectors : List["SequenceVector"],
+                           metric='euclidean'):
+        """
+        Convert a SequenceVector object to a DistanceMatrix object.
+
+        Parameters
+        ----------
+        sequence_vectors : iterable of SequenceVector objects
+            An iterable of SequenceVector objects.
+        metric : str, optional
+            The distance metric to use. Must be a valid metric for
+            `scipy.spatial.distance.pdist`.
+
+        Returns
+        -------
+        DistanceMatrix
+            A DistanceMatrix object.
+
+        See Also
+        --------
+        DistanceMatrix
+        """
+        data = SequenceVector.to_numpy(sequence_vectors)
+        ids = [str(pv) for pv in sequence_vectors]
+        dm = squareform(pdist(data, metric))
+        return DistanceMatrix(dm, ids=ids)
+
+    @staticmethod
+    def to_ordination(sequence_vectors : List["SequenceVector"]):
+        """
+        Convert a list of SequenceVector objects to an Ordination object.
+
+        Parameters
+        ----------
+        sequence_vectors : iterable of SequenceVector objects
+            An iterable of SequenceVector objects.
+
+        Returns
+        -------
+        OrdinationResults
+            An Ordination object.
+
+        See Also
+        --------
+        OrdinationResults
+        """
```

Comment:
It may be nice, in a separate PR, to refactor and move this logic to `skbio.stats.ordination` -- it seems generally useful to perform SVD -> `OrdinationResults`

Reply:
Yes! I find myself often recycling the SVD / biplot code